authz/src/main/java/io/grpc/authz/AuthorizationServerInterceptor.java

```diff
@@ -31,14 +31,20 @@
 import java.util.List;
 
 /**
- * Authorization server interceptor for static policy.
+ * Authorization server interceptor for static policy. The class will get
+ * gRPC Authorization policy as a JSON string during initialization. This
+ * policy will be translated to Envoy RBAC policies and used to initialize
+ * GrpcAuthorizationEngine objects to make authorization decisions. It
```

Comment:
Don't mention GrpcAuthorizationEngine; that is an implementation detail. "to Envoy RBAC policies to make authorization decisions."

Reply:
Sounds good. Updated comment.